### PR TITLE
fix typo.

### PR DESCRIPTION
**Pull request by dalanlan**
Signed-off-by: He Simei hesimei@zju.edu.cn

### CODE REVIEW

File: docs/sources/articles/basics.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ page_keywords: Examples, Usage, basic commands, docker, documentation, examples
 # First steps with Docker
 
 This guide assumes you have a working installation of Docker. To check
-your Docker install, run the following command:
+whether your Docker has been installed, run the following command:
```
Comment (michael-k) [inline docs/sources/articles/basics.md]: - your

File: docs/sources/articles/basics.md [hunk 2]
```diff
@@ -35,21 +35,19 @@ and download it from [Docker Hub](https://hub.docker.com) to a local
 image cache.
 
 > **Note**:
-> When the image has successfully downloaded, you will see a 12 character
+> When the image has been successfully downloaded, you will see a 12 character
```
Comment (moxiegirl) [inline docs/sources/articles/basics.md]: When the image is successfully downloaded, you see...

image

File: docs/sources/articles/basics.md [hunk 3]
```diff
@@ -35,21 +35,19 @@ and download it from [Docker Hub](https://hub.docker.com) to a local
...
 {{ include "no-remote-sudo.md" }}
 
 ## Running an interactive shell
 
-    # Run an interactive shell in the ubuntu image,
-    # allocate a tty, attach stdin and stdout
-    # To detach the tty without exiting the shell,
-    # use the escape sequence Ctrl-p + Ctrl-q
-    # note: This will continue to exist in a stopped state once exited (see "docker ps -a")
+    # Run an interactive shell in the ubuntu image, allocate a tty, attach stdin and stdout.
```
Comment (moxiegirl) [inline docs/sources/articles/basics.md]: This really needs to be text not comments.

--> https://gist.github.com/moxiegirl/f7e77a4c04743469d3f9

To run an interactive shell in the Ubuntu image:

    $ docker run -i -t ubuntu /bin/bash		     $ docker run -i -t ubuntu /bin/bash
  
The `-i` flag starts an interactive container. The `-t` flag creates a pseudo-TTY that attaches `stdin` and `stdout`.

File: docs/sources/articles/basics.md [hunk 4]
```diff
@@ -164,13 +162,13 @@ TCP and a Unix socket
 Save your containers state to an image, so the state can be
 re-used.
 
-When you commit your container only the differences between the image
+When you commit your container, only the differences between the image
```
Comment (moxiegirl) [inline docs/sources/articles/basics.md]: The original paragraph has something missing.  This is more succinct and fixes the problem:

--->

When you commit your container, Docker only stores the diff (difference) between the source image and the current state of the container's 		 image. To list images you already have, use the `docker images` command.

File: docs/sources/articles/basics.md (hunk 1 above)
Comment (moxiegirl) [inline docs/sources/articles/basics.md]: To verify Docker is installed, use the following command:

**Comment by moxiegirl** [comment]
@dalanlan thanks for the contribution. A few fixes please and it should be good-to-go.  (I love your icon. That was a cute cartoon.)

**Comment by dalanlan** [comment]
Updated @moxiegirl 
Glad that you love Baymax:)

**Comment by moxiegirl** [comment]
@dalanlan Good job.



 LGTM from me. Need one more from @docker/docs-owners

**Comment by thaJeztah** [comment]
thanks @dalanlan! LGTM


(and a nice avatar/movie indeed!)

**Pull request merged**